mscore/instrwidget.cpp

```diff
@@ -622,15 +622,15 @@ int InstrumentsWidget::findPrvItem(PartListItem* pli, int number)
       const int last = number ? number - 1 : partiturList->topLevelItemCount();
       QTreeWidgetItem* item = nullptr;
       for (int idx = 0; --number && (item = partiturList->topLevelItem(idx)); ++idx) {
-            PartListItem* pli = (PartListItem*)item;
```

Comment:
What is the problem this rename is solving? Just curious.

Reply:
MSVC complains the variable name to be the same as the function's parameter

Reply:
MSVC doesn't like shadowing then. Something to keep in mind.
By the way, it might be solved also by the new PR I'm creating for ScoreOrdering (is related to it).